local-running/local-config.yml

```diff
@@ -0,0 +1,45 @@
+# This config file is intended for use in development, when running the Verify
+# Service Provider as part of a locally running federation. It includes usages
+# of advanced configuration options which should not be necessary for relying
+# parties who are running against compliance tool, or the hub integration or
+# live environments.
+
+server:
+  type: simple
+  applicationContextPath: /
+  adminContextPath: /admin
+  connector:
+    type: http
+    port: ${VSP_PORT:-50400}
+
+logging:
+  level: ${LOG_LEVEL:-INFO}
+  appenders:
+    - type: console
+    - type: file
+      currentLogFilename: logs/verify-service-provider.log
+      archivedLogFilenamePattern: logs/verify-service-provider.log.%d.gz
+
+clockSkew: ${CLOCK_SKEW:-PT5s}
+
+serviceEntityIds: ${SERVICE_ENTITY_IDS:-["http://dev-rp.local/SAML2/MD"]}
```

Comment:
Are we not removing defaults from this config ?

Reply:
We could for consistency - it might mean making the verify-local-startup script more complicated, but probably not by much. But this is the new file specifically for starting locally (importantly, not a file which will be used by relying parties, and it shouldn't be in the distZip), so it doesn't run up against the rp-config-having-defaults-in issue.

Reply:
That said, this one is defined in verify-local-startup, so I'll take it out. The environment I think is the one which isn't (we're overriding most things from that in this config file anyway).